internal/server/backgroundjobs.go

```diff
@@ -7,21 +7,14 @@ import (
 
 	"github.com/infrahq/infra/internal/logging"
 	"github.com/infrahq/infra/internal/server/data"
-	"github.com/infrahq/infra/internal/server/jobs"
 )
 
 // BackgroundJobFunc is the interface for running periodic background jobs, like
 // garbage collection. Errors and panics from the job will be logged by
 // jobWrapper, and will not stop the goroutine running the job.
 // The job should exit when ctx is cancelled. Unlike most transactions, the
 // transaction passed to this job will not have an OrganizationID.
-type BackgroundJobFunc func(ctx context.Context, tx *data.Transaction) error
```

Comment:
I also removed the `context.Context` argument for a few reasons:
1. we don't currently use it
2. it could be misleading because the `tx` has a context itself already
3. by removing it we can use the `data` functions directly, instead of wrapping them